tests/System/expected/test___MarketingCampaignsReporting.getKeywordContentFromNameId_month.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<result />
```

Comment:
Is there a way to make this test have data in it?

Reply:
Actually it's on purpose to have no results here, as we send an invalid subtable id. could also add a test that actually has some data